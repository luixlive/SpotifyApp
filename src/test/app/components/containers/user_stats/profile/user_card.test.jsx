// TODO: test this
it('', () => expect(1).toBe(1));
