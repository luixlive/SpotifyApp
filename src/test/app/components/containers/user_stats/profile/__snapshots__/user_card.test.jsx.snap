// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Components - User Card Snapshots renders mobile and image invalid 1`] = `
<div
  className="ui one column grid"
>
  <div
    className="sixteen wide column"
  >
    <div>
      <div
        className="ui hidden divider"
      />
      <i
        aria-hidden="true"
        className="user circle outline huge icon"
      />
      <div
        className="ui hidden divider"
      />
    </div>
    <div
      className="ui fluid card"
      href={undefined}
      onClick={[Function]}
    >
      <div
        className="center aligned content"
      >
        <div
          className="header"
        >
          <h2
            className="ui header"
          >
            
          </h2>
        </div>
        <div
          className="meta"
        >
          <h4
            className="sub header"
          >
            
             (
            <a
              href=""
              rel="noopener noreferrer"
              target="_blank"
            >
              
            </a>
            )
          </h4>
        </div>
      </div>
      <div
        className="extra center aligned content"
      >
        0
         followers
      </div>
    </div>
  </div>
</div>
`;

exports[`App Components - User Card Snapshots renders mobile and image valid 1`] = `
<div
  className="ui one column grid"
>
  <div
    className="sixteen wide column"
  >
    <img
      className="ui small circular centered image"
      onError={[Function]}
      src="url"
    />
    <div
      className="ui fluid card"
      href={undefined}
      onClick={[Function]}
    >
      <div
        className="center aligned content"
      >
        <div
          className="header"
        >
          <h2
            className="ui header"
          >
            
          </h2>
        </div>
        <div
          className="meta"
        >
          <h4
            className="sub header"
          >
            
             (
            <a
              href=""
              rel="noopener noreferrer"
              target="_blank"
            >
              
            </a>
            )
          </h4>
        </div>
      </div>
      <div
        className="extra center aligned content"
      >
        0
         followers
      </div>
    </div>
  </div>
</div>
`;

exports[`App Components - User Card Snapshots renders no mobile and image invalid 1`] = `
<div
  className="ui three column grid"
>
  <div
    className="four wide column"
  />
  <div
    className="eight wide column"
  >
    <div>
      <div
        className="ui hidden divider"
      />
      <i
        aria-hidden="true"
        className="user circle outline massive icon"
      />
      <div
        className="ui hidden divider"
      />
    </div>
    <div
      className="ui fluid card"
      href={undefined}
      onClick={[Function]}
    >
      <div
        className="center aligned content"
      >
        <div
          className="header"
        >
          <h1
            className="ui header"
          >
            
          </h1>
        </div>
        <div
          className="meta"
        >
          <h3
            className="sub header"
          >
            
             (
            <a
              href=""
              rel="noopener noreferrer"
              target="_blank"
            >
              
            </a>
            )
          </h3>
        </div>
      </div>
      <div
        className="extra center aligned content"
      >
        0
         followers
      </div>
    </div>
  </div>
  <div
    className="four wide column"
  />
</div>
`;

exports[`App Components - User Card Snapshots renders no mobile and image valid 1`] = `
<div
  className="ui three column grid"
>
  <div
    className="four wide column"
  />
  <div
    className="eight wide column"
  >
    <img
      className="ui small circular centered image"
      onError={[Function]}
      src="url"
    />
    <div
      className="ui fluid card"
      href={undefined}
      onClick={[Function]}
    >
      <div
        className="center aligned content"
      >
        <div
          className="header"
        >
          <h1
            className="ui header"
          >
            
          </h1>
        </div>
        <div
          className="meta"
        >
          <h3
            className="sub header"
          >
            
             (
            <a
              href=""
              rel="noopener noreferrer"
              target="_blank"
            >
              
            </a>
            )
          </h3>
        </div>
      </div>
      <div
        className="extra center aligned content"
      >
        0
         followers
      </div>
    </div>
  </div>
  <div
    className="four wide column"
  />
</div>
`;
